phone/app/src/main/res/xml/wear.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/22f3a586e71cf8d173d7c2e711ba5a3f463d2e79/phone/app/src/main/res/xml/wear.xml#L2"></span>The resource <code>R.xml.wear</code> appears to be unused</td>
    </tr>
  </tbody>
</table>